docs/usage-serializers.md

```diff
@@ -137,6 +137,49 @@ class OwnerSerializer(SerializerExtensionsMixin, ModelSerializer):
 Setting `id_source=False` results in no ID field being included.
 
 
+### Writable ForeignKey relationships
+In some cases it may be appropriate to allow ForeignKey fields to be set during
+a create or an update via your API. To create a ForeignKey relation that is
+both expandable and writable, set `read_only=False` in its definition.
+
+```py
+# serializers.py
+class OwnerSerializer(SerializerExtensionsMixin, ModelSerializer):
+    class Meta:
+        model = models.Owner
+        fields = ('id', 'name')
+        expandable_fields = dict(
+            organization=dict(
+                serializer=OrganizationSerializer,
+                read_only=False
+            )
+        )
+```
+
+As usual, the above serializer will have an `organization_id` field by default.
+To set or alter the organization of an owner, pass a value to this field:
+
+```py
+serializer = OwnerSerializer(data=dict(name="Elliot", organization_id=2))
```

Comment:
👍 , this might be clearer (and more consistent with other documentation) by using this:

```js
>>> POST /owner/2/
{
  "name": "Elliot",
  "organization_id": 2
}
```